distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -693,6 +708,161 @@ public void persistObjectToWorkspace() throws Exception {
         persistToWorkspace(100);
     }
 
+    //@Ignore
+    @Test
+    public void testValidationEnforced() throws Exception {
+        getServiceManager().startFeature(true, "catalog-plugin-metacard-validation");
+
+        // Update metacardMarkerPlugin config with enforcedMetacardValidators
+        Configuration metacardMarker = configAdmin
+                .getConfiguration("ddf.catalog.metacard.validation.MetacardValidityMarkerPlugin",
+                        null);
+        Dictionary<String, Object> properties = new Hashtable<>();
+        List<String> enforcedMetacardValidators = new ArrayList<>();
+        enforcedMetacardValidators.add("sample-validator");
+        properties.put("enforcedMetacardValidators", enforcedMetacardValidators);
+        metacardMarker.update(properties);
+
+        String id1 = ingestXmlFromResource("/metacard1.xml");
+        String id2 = ingestXmlFromResource("/metacard2.xml", false);
+
+        // Search for all entries, implicit "valid-state=true" should get added by MetacardValidityCheckerPlugin
```

Comment:
Please update your comments.  They are no longer correct with the new multivalued attribute approach.